src/config/routes.ts

```diff
@@ -1,3 +1,4 @@
+export const BASE_URL = process.env.NEXT_PUBLIC_BASE_URL ?? 'https://expenses.makerdao.network';
 export const HOME_ROUTE = '/';
```

Comment:
I know that its not relate with the pr, but i think that you can remove unnecessary router in the file

Reply:
Sure, I will